src/components/info-card.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+interface InfoCardProps {
+    children: React.ReactNode;
+}
+
+const InfoCard: React.FC<InfoCardProps> = ({ children }) => {
+    return (
+        <div className='bg-white rounded-[20px] border-[1px] border-[#E8E8E8] min-w-[300px]'>
```

Comment:
We're trying to always use Tailwind values for widths, border, etc. Please use the closest default Tailwind value instead of `20px` (that would be `rounded-2xl` (16px) or `rounded-3xl` (24px)).

`border-[1px]` is the same as just `border`

`border-[#E8E8E8]`: let's try not to have hard-coded color values, either use a gray already defined in `tailwind.config.ts` or add a new color to the config:

```
...
colors: {
        primary: "#ED7950",
        background: "#FAFAFA",
        stroke: "#D4D4D8",
        black: "#27272A",
        gray: "#94959C",
        stone: "#27272A",
        blue: "#5069ED",
      },
...
```